app/src/main/java/fr/free/nrw/commons/explore/map/ExploreMapMediaDataSource.kt

```diff
@@ -0,0 +1,26 @@
+package fr.free.nrw.commons.explore.map
+
+import android.util.Log
+import fr.free.nrw.commons.Media
+import fr.free.nrw.commons.explore.depictions.search.LoadFunction
+import fr.free.nrw.commons.explore.paging.LiveDataConverter
+import fr.free.nrw.commons.explore.paging.PageableBaseDataSource
+import fr.free.nrw.commons.media.MediaClient
+import fr.free.nrw.commons.media.WikidataMediaClient
+import javax.inject.Inject
+
+class ExploreMapMediaDataSource @Inject constructor(
+    liveDataConverter: LiveDataConverter,
+    private val mediaClient: MediaClient,
+    val isFromSearchActivity: Boolean
```

Comment:
This dependency isn't defined so the dagger does not know how to inject it here.